spec.html

```diff
@@ -12767,18 +12791,25 @@ <h1>Runtime Semantics: PropertyDefinitionEvaluation</h1>
         <emu-alg>
           1. Let _propKey_ be the result of evaluating |PropertyName|.
           1. ReturnIfAbrupt(_propKey_).
-          1. If IsAnonymousFunctionDefinition(|AssignmentExpression|) is *true*, then
+          1. If this |PropertyDefinition| is contained within a |Script| which is being evaluated for JSON.parse (see step <emu-xref href="#step-json-parse-parse"></emu-xref> of <emu-xref href="#sec-json.parse">JSON.parse</emu-xref>), then
```

Comment:
it occurs to me we could take a leaf from eval and do something like "if the [[Function]] slot of the second-to-top execution context is %JSON.parse%"

Reply:
1. Let callerContext be the second to top element of the execution context stack.
1. Let calleeContext be the running execution context.
1. If calleeContext.[[Function]] is null and callerContext.[[Function]] is %JSON.parse%, then
  a. Let isProtoSetter be false.